packages/fast-tooling-react/src/form/templates/template.control.standard.tsx

```diff
@@ -0,0 +1,97 @@
+ */
+class StandardControlTemplate extends AbstractControlTemplate<
+    StandardControlTemplateProps &
+        ManagedClasses<StandardControlTemplateClassNameContract>,
+    {}
+> {
+    public static defaultProps: Partial<
+        StandardControlTemplateProps &
+            ManagedClasses<StandardControlTemplateClassNameContract>
+    > = {
+        context: ControlContext.default,
+        softRemove: true,
+        managedClasses: {},
+    };
+
+    public render(): React.ReactNode {
+        const {
+            standardControlTemplate,
+            standardControlTemplate__disabled,
+            standardControlTemplate_controlRegion,
+            standardControlTemplate_control,
+            standardControlTemplate_controlLabelRegion,
+            standardControlTemplate_controlLabel,
+            standardControlTemplate_constValueIndicator,
+            standardControlTemplate_defaultValueIndicator,
+            standardControlTemplate_badge,
+            standardControlTemplate_softRemove,
+            standardControlTemplate_softRemoveInput,
+            standardControlTemplate_invalidMessage,
+        }: StandardControlTemplateClassNameContract = this.props.managedClasses;
+
+        return (
+            <div
+                className={classNames(standardControlTemplate, [
+                    standardControlTemplate__disabled,
+                    this.props.disabled,
+                ])}
+            >
+                <div className={standardControlTemplate_controlRegion}>
+                    <div className={standardControlTemplate_control}>
+                        <div className={standardControlTemplate_controlLabelRegion}>
+                            <label
+                                htmlFor={this.props.dataLocation}
+                                className={standardControlTemplate_controlLabel}
+                            >
+                                {this.props.label}
+                            </label>
+                            {this.renderConstValueIndicator(
+                                standardControlTemplate_constValueIndicator
+                            )}
+                            {this.renderDefaultValueIndicator(
+                                standardControlTemplate_defaultValueIndicator
+                            )}
+                            {this.renderBadge(standardControlTemplate_badge)}
+                        </div>
+                        {this.renderControlInDefaultContext()}
+                    </div>
+                    <div className={standardControlTemplate_softRemove}>
+                        {this.renderSoftRemove(standardControlTemplate_softRemoveInput)}
+                    </div>
+                </div>
+                {this.renderControlInFillContext()}
+                {this.renderInvalidMessage(standardControlTemplate_invalidMessage)}
+            </div>
+        );
+    }
+
+    private renderControlInDefaultContext(): React.ReactNode {
```

Comment:
You don't really need two functions for this - maybe just supply the `ControlContext` success case as a param?